changelog/147.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+agent,product,op: rename seeker to Op.
```

Comment:
What if we changed the component to just "op" instead of "agent,product,op"? I think that it will be easier for readers to consume if each entry only applies to a single component. (We should probably discuss what the list of components would be - maybe we could put them into a PR template so it's front-and-center whenever a new PR is opened.)